ocean_provider/validation/provider_requests.py

```diff
@@ -266,10 +265,11 @@ def rules(self):
 class FileInfoRequest(CustomJsonRequest):
     def rules(self):
         return {
-            "type": ["required_without:did", "in:ipfs,url"],
+            "type": ["required_without:did", "in:ipfs,url,arweave"],
```

Comment:
I'm thinking of creating a list called ALLOWED_FILE_TYPES which we can use in instance generation and here as well, to provide a single entrypoint for modifying. Would you like to do that?

I usually guide myself by a rule of "it's ok to do something twice, but the third time I'm automating it"

Reply:
Done.